### ISSUE TITLE
Interpolation-only Expression Deprecation

### ISSUE THREAD

**Issue by codygreen** [f5devcentral/terraform-aws-bigip-module#15]
When using this module I get the following warning:

```bash
Warning: Interpolation-only expressions are deprecated

  on .terraform/modules/bigip/main.tf line 196, in data "aws_ami" "f5_ami":
 196:     values = ["${var.f5_ami_search_name}"]
```

This needs to be changed to:
```hcl
values = format("%s", var.f5_ami_search_name)
```

**Comment by nmenant** [comment]
Tracking this internally INFRAANO-164

**Comment by mjmenger** [comment]
@codygreen can't this be fixed with 
```
values = var.f5_ami_search_name
```

**Comment by papineni87** [comment]
Yes, we can fix it with above expression also and tested it.

I  had raised a PR, which will take care all of the depreciation issues  in the aws module ( https://github.com/f5devcentral/terraform-aws-bigip-module/pull/24 )

**Comment by papineni87** [comment]
Issue is fixed in latest release v0.9.1, please reopen this issue if you see it again
